dll/win32/shimgvw/shimgvw.c

```diff
@@ -959,6 +977,58 @@ ImageView_OnSize(HWND hwnd, UINT state, INT cx, INT cy)
     }
 }
 
+static LRESULT CALLBACK
```

Comment:
So that you might use STDCALL.

Reply:
I doubt this will make the compiler generate better code,
and in cases like this making an assumption like that can even cause the compiler to generate worse code.

Considering this is not going to be a hot path, those optimizations (and especially, optimizations based on guesses) are better not added.